prometheus_client/exposition.py

```diff
@@ -159,7 +159,38 @@ def _get_best_family(address, port):
     return family, sockaddr[0]
 
 
-def start_wsgi_server(port: int, addr: str = '0.0.0.0', registry: CollectorRegistry = REGISTRY) -> None:
+def _get_ssl_ctx(
+        certfile: str,
+        keyfile: str,
+        protocol: int,
+        cafile: Optional[str] = None,
+        insecure_skip_verify: bool = False,
+) -> ssl.SSLContext:
+    """Load context supports SSL."""
+    ssl_cxt = ssl.SSLContext(protocol=protocol)
+    if cafile is not None:
+        ssl_cxt.load_verify_locations(cafile)
+    else:
+        ssl_cxt.load_default_certs()
+
+    if insecure_skip_verify:
+        ssl_cxt.check_hostname = False
+        ssl_cxt.verify_mode = ssl.CERT_NONE
+
+    ssl_cxt.load_cert_chain(certfile=certfile, keyfile=keyfile)
+    return ssl_cxt
+
+
+def start_wsgi_server(
+        port: int,
+        addr: str = '0.0.0.0',
+        certfile: str = '',
+        keyfile: str = '',
+        cafile: Optional[str] = None,
+        protocol: int = ssl.PROTOCOL_TLS_SERVER,
+        insecure_skip_verify: bool = False,
+        registry: CollectorRegistry = REGISTRY,
```

Comment:
Putting the new parameters before the previously existing `registry` parameter is an incompatibility for users that pass positional arguments. I think we should stay compatible if possible, i.e. add the new parameters after the previously existing ones.

I think a `*` before the new parameters would also not hurt to require passing them as kwargs.

Reply:
Was done. Thanks!